README.md

```diff
@@ -46,7 +46,10 @@ The list of items you’ll need is also included below:
 
 ## Setup and configuration
 
-Use of balenaDash requires that you allocate more memory to the GPU. This is achieved by adding (or editing the existing) the **Device configuration variable** `RESIN_HOST_CONFIG_gpu_mem`, for this project we recommend setting it to `396`.
+Use of balenaDash requires that you allocate more memory to the GPU. This is achieved by adding (or editing the existing) the **Device configuration variable** `RESIN_HOST_CONFIG_gpu_mem`, for this project we recommend setting it to `128`.
+
+Or this project can be deployed to a new balenaCloud application in one click using the button below:
```

Comment:
What I am thinking is we can move this above, and after it state that when deploying manually, they need to set the GPU mem to some value.

Reply:
@sradevski new version pushed. Hopefully this looks OK to everyone.